httprequesthandler_test.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/openfaas-incubator/of-watchdog/config"
+)
+
+func TestHTTPRequestHandler_WatchdogPassesVerbToUpstream(t *testing.T) {
+
+	verbs := []string{http.MethodGet, http.MethodPut, http.MethodPut, http.MethodDelete}
+
+	wc := config.WatchdogConfig{
+		FunctionProcess: "sleep 10",
```

Comment:
This would make the test run for 10 seconds which is less than ideal. See how it's done in the watchdog tests in the faas repo https://github.com/openfaas/faas/

Reply:
This was removed from the PR.